nixos/modules/services/web-servers/garage.nix

```diff
@@ -75,7 +85,16 @@ in
       source = configFile;
     };
 
-    environment.systemPackages = [ cfg.package ]; # For administration
+    # For administration
+    environment.systemPackages = if cfg.environmentFile != null
```

Comment:
suggestion: I don't think we need to branch in this place, we should always pass the wrapper, worst case, there's no environment file and it fallback gracefully to an exec to the binary.